lib/rules/no-invalid-aria-attributes.js

```diff
@@ -0,0 +1,170 @@
+'use strict';
+
+const ariaQuery = require('aria-query');
+
+const Rule = require('./_base');
+
+const VALID_ARIA_ATTRIBUTES = ariaQuery.aria;
+
+function createInvalidAriaAttributeMessage(name) {
+  return `${name} is not a valid ARIA attribute.`;
```

Comment:
It's not clear what is meant here by "not valid". I think this is trying to say that the ARIA attribute is unknown or unrecognized / non-existent. Can we instead say something like:
```suggestion
  return `${name} is an unrecognized ARIA attribute.`;
```